buildSrc/src/main/groovy/org/elasticsearch/gradle/plugin/PluginPropertiesExtension.groovy

```diff
@@ -56,17 +56,38 @@ class PluginPropertiesExtension {
 
     /** A license file that should be included in the built plugin zip. */
     @Input
```

Comment:
I don't think this can be @Input if it is private? It needs the default visibility to allow groovy/gradle to wrap in such a way that changes can be tracked.

Reply:
good catch, thanks!
The annotation needs to be on the getter, then we can keep this private to make sure it goes trough the setter.